.gitignore

```diff
@@ -6,3 +6,4 @@ venv
 build
 dist/
 *.orig
+.idea
```

Comment:
I use pycharm too :)  Maybe we all should.  but some find it to not be relatable to add to a project's `.gitignore`.

check out [global ignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
In fact, it always annoys me to add `.idea` to `.gitignore` or just ignored myself. So this is a very useful advice. Thanks